app/routines/find_book.rb

```diff
@@ -0,0 +1,50 @@
+# Finds an approved book with the given UUID in the ABL that can be loaded
+class FindBook
```

Comment:
Could this be used in `lib/content.rb` as well?

Reply:
That one deals with all the books at once. Maybe it could be extracted into a routine that takes a list of books, then this one could use that. I'm not sure it's worth the effort since Exercises is probably going away eventually anyway.